### PR TITLE
Extend app.config.json for storage explorer link template

### PR DESCRIPTION
**Pull request by marcgs**
Extend app.config.json as required for https://github.com/agera-edc/EDCDataDashboard/pull/12

### CODE REVIEW

File: deployment/terraform/main.tf [hunk 1]
```diff
@@ -163,6 +163,7 @@ resource "azurerm_container_group" "webapp" {
           "dataManagementApiUrl" = "http://${azurerm_container_group.edc.fqdn}:${local.edc_management_port}/api/v1/data"
           "catalogUrl"           = "http://${azurerm_container_group.edc.fqdn}:${local.edc_default_port}/api/federatedcatalog"
           "storageAccount"       = azurerm_storage_account.inbox.name
+          "storageExplorerLinkTemplate" = "storageexplorer://v=1&accountid=/subscriptions/${data.azurerm_subscription.current_subscription.id}/resourceGroups/${azurerm_resource_group.participant.name}/providers/Microsoft.Storage/storageAccounts/{{account}}&subscriptionid=${data.azurerm_subscription.current_subscription.id}&resourcetype=Azure.BlobContainer&resourcename={{container}}",
```
Comment (algattik) [inline deployment/terraform/main.tf]: Nit: azurerm_resource_group.participant.id to shorten this?

**Review by algattik — APPROVED** [review]

File: deployment/terraform/main.tf (hunk 1 above)
Comment (marcgs) [reply deployment/terraform/main.tf]: does it contain the same value?

**Review by cpeeyush — APPROVED** [review]

File: deployment/terraform/main.tf (hunk 1 above)
Comment (algattik) [reply deployment/terraform/main.tf]: I expect it would contain everything starting at "/subscriptions"

Comment (marcgs) [reply deployment/terraform/main.tf]: Done, it works well!

File: deployment/terraform/main.tf [hunk 2]
```diff
@@ -163,6 +163,7 @@ resource "azurerm_container_group" "webapp" {
           "dataManagementApiUrl" = "http://${azurerm_container_group.edc.fqdn}:${local.edc_management_port}/api/v1/data"
           "catalogUrl"           = "http://${azurerm_container_group.edc.fqdn}:${local.edc_default_port}/api/federatedcatalog"
           "storageAccount"       = azurerm_storage_account.inbox.name
+          "storageExplorerLinkTemplate" = "storageexplorer://v=1&accountid=${azurerm_resource_group.participant.id}/providers/Microsoft.Storage/storageAccounts/{{account}}&subscriptionid=${data.azurerm_subscription.current_subscription.subscription_id}&resourcetype=Azure.BlobContainer&resourcename={{container}}",
```
Comment (algattik) [inline deployment/terraform/main.tf:166]: `terraform fmt` bitte

Comment (marcgs) [reply deployment/terraform/main.tf:166]: Not sure to understand what you mean. Can be done as a subsequent improvement?

**Pull request merged**